src/components/atoms/Button/style.css

```diff
@@ -0,0 +1,46 @@
+.carousel__button {
```

Comment:
write isolated styles for components inside component folder not related to any other component